src/modules/Home/Home.css

```diff
@@ -0,0 +1,50 @@
+.flex-row-container {
```

Comment:
I think either here or the .flex-column-container, you can do 
flex: 1 1 33%
for a row of three columns/images.
More details: https://developer.mozilla.org/en-US/docs/Web/CSS/flex